src/sidebar/util/default-keys.js

```diff
@@ -0,0 +1,7 @@
+/**
```

Comment:
Hokay, I know this util is sorta silly because it only has one entry, but I the keyname-mapping is needed by both the `store/index` module and the `persistedDefaults` service, so I wanted them to exist somewhere neutral and shared.